pages/mesosphere/dcos/services/kafka/2.7.0-2.3.0/started/index.md

```diff
@@ -0,0 +1,617 @@
+
+```json
+    {
+        "kafka": {
+            "log_retention_hours": 100
+        }
+    }
+```
+
+These same values are also represented as environment variables for the scheduler in the form `KAFKA_OVERRIDE_LOG_RETENTION_HOURS` and may be modified through the DC/OS web interface and deployed during a rolling upgrade as described in [changing configuration at runtime](/mesosphere/dcos/services/kafka/2.6.0-2.2.1/started/#changing-configuration-at-runtime).
+
+<a name="disk-type"></a>
+## Disk Type
+
+The type of disks that can be used for storing broker data are: `ROOT` (default) and `MOUNT`.  The type of disk may only be specified at install time.
+
+* `ROOT`: Broker data is stored on the same volume as the agent work directory. Broker tasks will use the configured amount of disk space.
+* `MOUNT`: Broker data will be stored on a dedicated volume attached to the agent. Dedicated MOUNT volumes have performance advantages and a disk error on these MOUNT volumes will be correctly reported to Kafka.
+
+Configure Kafka service to use dedicated disk volumes, as follows:
+* **DC/OS cli options.json**:
+
+```json
+    {
+        "brokers": {
+            "disk_type": "MOUNT"
+        }
+    }
+```
+
+* **DC/OS web interface**: Set the environment variable `DISK_TYPE`: `MOUNT`
+
+When configured to `MOUNT` disk type, the scheduler selects a disk on an agent whose capacity is equal to or greater than the configured `disk` value.
+
+## JVM Heap Size
+
+Kafka service allows configuration of JVM Heap Size for the broker JVM process. Use the following configuration options:
+
+```json
+    {
+        "brokers": {
+            "heap": {
+                "size": 2000
+            }
+        }
+    }
+```
+
+* **DC/OS web interface**: Set the environment variable `BROKER_HEAP_MB`: `2000`
+
+<p class="message--note"><strong>NOTE: </strong>The total memory allocated for the Mesos task is specified by the `BROKER_MEM` configuration parameter. The value for `BROKER_HEAP_MB` should not be greater than `BROKER_MEM` value. Also, if `BROKER_MEM` is greater than `BROKER_HEAP_MB` then the Linux operating system will use `BROKER_MEM` - `BROKER_HEAP_MB` for [PageCache](https://en.wikipedia.org/wiki/Page_cache).</p>
+
+## Alternate ZooKeeper
+
+By default the Kafka framework uses the ZooKeeper ensemble made available on the Mesos masters of a DC/OS cluster. You can configure an alternate ZooKeeper at install time.
+
+Use the following configuration options:
+
+* **DC/OS CLI options.json**:
+
+```json
+    {
+        "kafka": {
+            "kafka_zookeeper_uri": "zookeeper.marathon.autoip.dcos.thisdcos.directory:2181"
+        }
+    }
+```
+
+This configuration option cannot be changed after installation.
+
+## Recovery and Health Checks
+
+You can enable automated replacement of brokers and configure the circumstances under which they are replaced.
+
+### Enable Broker Replacement
```

Comment:
Hm, this is entirely separate from the advanced health checks you've added, right?

Reply:
Yes